src/ortc.cpp

```diff
@@ -920,6 +920,7 @@ namespace mediasoupclient
 			auto opusMaxPlaybackRateIt     = params.find("opusMaxPlaybackRate");
 			auto opusMaxAverageBitrateIt   = params.find("opusMaxAverageBitrate");
 			auto opusPtimeIt               = params.find("opusPtime");
+			auto opusCbrIt                 = params.find("opusCbr");
```

Comment:
Can you please put the `cbr` logic after  the `dtx` one among the PR? They belong to the same logical group of options.

Thanks.

Reply:
@jmillan moved.